app/backend/core/authentication.py

```diff
@@ -32,7 +32,7 @@ def __str__(self) -> str:
 
 
 class AuthenticationHelper:
-    scope: str = "https://graph.microsoft.com/.default"
+    scope: str = "https://search.azure.com/.default"
```

Comment:
This change may make it slightly harder for people to extend the sample for arbitrary Graph API access, like to read emails, as I once demo'ed. Not sure who's actually doing that though. Just noting.

Reply:
I see - unfortunately, we cannot request both scopes here. We can give guidance to do this twice though in the custom code